api/app/schemas/fba.py

```diff
@@ -55,6 +55,29 @@ class HfiThresholdAreaByFuelType(BaseModel):
     area: float
 
 
+class HfiThreshold(BaseModel):
+    """ An HFI Classification threshold """
```

Comment:
Unused

Reply:
HfiThreshold is referenced in the `ClassifiedHfiThresholdFuelTypeArea` class, which is in turn being used in api/app/routers/fba.py